packages/core/src/components/pagination/pagination.tsx

```diff
@@ -49,30 +74,130 @@ export class Pagination implements ComponentInterface {
     const pages = Array.from({ length: this.totalPages }, (_value, i) => i + 1);
```

Comment:
Ik ben toch niet zo gelukkig met deze opzet, dit was eerder nodig omdat _alle_ pagina's ook toonde. Nu maken we bij 120 totalPages best een grote array, om vervolgens 110 van weg te gooien. 

Kunnen we niet op dit vlak wat doen? Door bijvoorbeeld op basis van `availablePositions` icm `currentPage` een Array te instantieren?

Reply:
Ik hoop dat het wat te begrijpen is, maar het is nogsteeds complex.

Reply:
We laten het even